### DISCUSSION TITLE
Archive/hide a calendar

### DISCUSSION THREAD

**Discussion by WilliamMayor** [Kozea/Radicale#1457] [Ideas]
I'd like to be able to archive calendars so that they no longer appear in my list of calendars in my client. I'd like to be able to un-archive them too.

Can anyone point me in the right direction? I'm guessing that I'll need a plugin that listens for some metadata to be set/unset and then does something. Maybe moves the collection into a separate archive directory?

I'd appreciate any help, thanks!

**Comment by chripo** [comment]
the easiest way would be to move the collections to an other folder.
did you try a rights rule, i think you should place it on top

```
[archived]
user = .*
collection = USER_NAME/colleciton-(foo|bar|baz)
permission =
```

**Comment by fwdekker** [comment]
As an alternative interpretation, archived calendars could be made read-only server-side, and would appear at the bottom of the list in the web interface with a special marker indicating its archived status. A bit similar to how GitHub handles archiving repos, I suppose.

**Comment by pbiering** [comment]
Potential solution would be adding a custom flag into `.Radicale.props` (https://github.com/Kozea/Radicale/wiki/Collection-Storage#file-radicaleprops) which prevents writing to a collection on server side and extending the WebUI to add/remove/change this flag by custom API call (https://github.com/Kozea/Radicale/wiki/Radicale-Custom-DAV-protocol-extensions).

Can one implement this and submitting a PR?
